docs/cluster-setup-guide/deploy-cluster/sysadmin-deploy-on-slurm/hpc-launching-architecture.rst

```diff
@@ -0,0 +1,370 @@
+=======================
+
+Temporary files (generated batch files, model python code, ``startup-hook.sh``, etc.), per-rank
+local directories in each container (``/tmp``, ``/run/determined/workdir``, etc.), and job log files
+are all stored under the cluster-wide directory specified via ``resource_manager.job_storage_root``.
+The lifetime of these files is limited to that of the submitted job, and are removed when job
+failure or completion is detected by Determined.
+
+#. Determined submits a job to the launcher.
+
+#. A directory tree is created by the launcher for the job.
+
+#. Files provided by Determined are staged.
+
+#. A batch script is generated to describe the required resource request (see Job Resource
+   Scheduling) from the workload manager and context (environment variables, and container launch
+   commands).
+
+#. The ownership of the files is transferred to the launching user (``sudo chmod``).
+
+#. A setuid image is executed to impersonate the launching user, which in turn executes the
+   sbatch/qsub command to submit the job.
+
+#. Upon being scheduled by the workload manager, the batch script is executed and tasks are launched on the allocated compute nodes.
+      #. One Determined environment image is started per node (``srun``, ``pbs_tmrsh``) using the
+         specified container runtime (Apptainer, Singularity, Podman, Enroot).
+
+      #. During container initialization, per-rank local directories are configured using soft links
+         (``/tmp``, ``/run/determined``), and the container stdout/stderr starts streaming output
+         directly to the Determined master.
+
+      #. For distributed training, horovodrun is invoked within the container to distribute work
+         within multiple processes across the containers.
+
+#. Once started, the job ID from the workload manager is displayed in the experiment log.
+
+#. Determined polls the launcher for job status waiting for completion.
+
+#. Determined requests the launcher to clean up the job. File ownership is transferred back to the
+   launcher (``sudo chmod``) which then deletes all temporary files and logs associated with the
+   job.
+
+*************************
+ Job Resource Scheduling
+*************************
+
+As noted earlier, when using the HPC launcher, Determined delegates all job scheduling and
+prioritization to the HPC workload manager (Slurm/PBS). Resource requests are generated into the
+submitted batch file derived from the following Determined configuration attributes.
+
+Cluster Configuration
+=====================
+
+The :ref:`resource_manager <cluster-configuration-slurm>` section of the cluster configuration
+contributes the following resource scheduling configuration. See the :ref:`slurm/pbs section
+<cluster-configuration-slurm>` of the cluster configuration reference for the full list of
+configuration options.
+
+-  ``slot_type``: The default slot type (``cuda``, ``rocm``, ``cpu``) assumed when users request
```

Comment:
Why "assumed"?  If you specify a value, it is not assumed.  Perhaps "that is used" would be better?

Reply:
This is the existing wording from the cluster reference.  It doesn't force the overall slot_type -- that comes from the resource pool  -- it does specify cuda vs rocm for the resource pool if there are GPUs, or I suppose it limits it to cpu if it would have otherwise chosen cuda.